src/main/java/uk/gov/register/resources/DataUpload.java

```diff
@@ -55,12 +56,13 @@ public DataUpload(ObjectReconstructor objectReconstructor, RegisterSerialisation
     @Path("/load")
     @Timed
     public void load(String payload) {
+        // TODO remove this endpoint
```

Comment:
I think this endpoint is described in the spec: https://openregister.github.io/specification/#minting so if we remove it we need to update the spec to correspond (CC @arnau) maybe best to remove this TODO for the time being?